analyses/create-corplots/01-process-data.Rmd

```diff
@@ -0,0 +1,270 @@
+  dplyr::rename(Kids_First_Biospecimen_ID = Tumor_Sample_Barcode) %>% # change the column name of the biospecimen column to match the column name from the histologies files
+  dplyr::mutate(VAF = t_alt_count / (t_ref_count + t_alt_count)) %>% # calculate VAF
+  select(Kids_First_Biospecimen_ID, Hugo_Symbol, Chromosome, Start_Position, Variant_Classification, VAF, HGVSc, HGVSp, HGVSp_Short) 
+
+## Read in genomic_assays_matched_time_points file and get the list of patients
+genomic_assays_matched_time_points_list <- readr::read_tsv(genomic_assays_matched_time_points, guess_max = 100000, show_col_types = FALSE) %>%
+                                      subset(select = c("Kids_First_Participant_ID", "descriptors"))
+  
+# Read file with TMB information
+tmb <- readr::read_tsv(tmb, guess_max = 100000, show_col_types = FALSE)
+```
+
+# Select patients with genomic assays
+
+```{r echo=TRUE}
+# We will look only into samples with genomic assays
+# We will use the list fo the patients from the "genomic_assays_matched_time_points_list" and merge to the pbta table
+# We need to count how many tumor samples we have per patient/time point, if any
+# We need to create a column to do that: match_id_multiple_samples
+genomic_assays_matched_time_points_list_merge <- genomic_assays_matched_time_points_list %>% 
+  left_join(pbta, by=c('Kids_First_Participant_ID')) %>%
+  filter(!(experimental_strategy == "RNA-Seq")) %>%
+  mutate(match_id_multiple_samples = paste(Kids_First_Participant_ID, sample_id, composition, sep = "_")) %>%
+  mutate(tumor_descriptor = case_when(grepl("Primary Tumor", tumor_descriptor) ~ "Diagnosis",
+                                 grepl("Initial CNS Tumor", tumor_descriptor) ~ "Diagnosis",
+                                 grepl('Progressive Disease Post-Mortem', tumor_descriptor) ~ 'Deceased', 
+                                 TRUE ~ tumor_descriptor)) %>%
+  mutate(Deceased = case_when(grepl("Deceased", descriptors) ~ "Deceased")) %>%
+  filter(Deceased == "Deceased") %>%
+  write_tsv(file.path(results_dir, "genomic_assays_matched_time_points_list_merge.tsv"))
+```
+
+# Autopsy samples 
+We will focus on patient cases containing autopsy samples and save these maf files separately.
+We will do this for different pairs.
+
+
+```{r maf_files_Deceased}
+#let us filter by Deceased samples
+maf_files_Deceased <- genomic_assays_matched_time_points_list_merge %>% 
+  left_join(snv, by=c('Kids_First_Biospecimen_ID')) %>%
+  filter(!is.na(Kids_First_Participant_ID)) %>%
+  write_tsv(file.path(scratch, "maf_files_Deceased.tsv"))
+```
+
+## Let's save the maf files per each category
+
+```{r save-maf-files}
+##########################################################################################
+# Storing all Dx-Deceased Sample pairs in a list so each file can be processed iteratively
+scratch_Dx_Dec <-
+  file.path(scratch, "Dx_Dec")
+if (!dir.exists(scratch_Dx_Dec)) {
+  dir.create(scratch_Dx_Dec)
+}
```

Comment:
when creating new directories, put these all at the top of the script, rather than within (and update the variable of the scratch directory to `scratch_dir`). Try to use lowercase convention for all of the directories